bowling_game.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+$LOAD_PATH.push('Users/andrespacheco/Documents/andres-pacheco/bowling-game-kata-pacheco-andres')
+
+load 'frame.rb'
+load 'game_roll.rb'
+
+# bowling game logic class
+class BowlingGame
+  def initialize
+    @pins_down_collection = [10, 2, 3, 0, 0, 4, 6, 0, 0, 3, 4, 10, 10, 9, 3, 8, 1]
```

Comment:
Is this a simulation of the game?

Reply:
Yes, I put the numbers fixed for simulate the turns values